.github/workflows/model-integration.yml

```diff
@@ -63,7 +63,6 @@ jobs:
           ./gradlew -p codegen/sdk-codegen build
           ./gradlew -p codegen/sdk-codegen stageSdks
           ./gradlew --stop
-          ./scripts/mergeModels.sh Sources/Services
```

Comment:
Invocations of `mergeModels.sh` are removed here and in other scripts where it is currently used.